_inc/lib/core-api/wpcom-fields/post-fields-publicize-connections.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+/**
+ * Add per-post Publicize Connection data.
+ *
+ * { # Post Object
+ *   ...
+ *   jetpack_publicize_connections: { # Defined below in this file. See schema for more detail.
+ *     id:           (string)  Connection unique_id
+ *     service_name: (string)  Service slug
+ *     display_name: (string)  User name/display name of user/connection on Service
+ *     enabled:      (boolean) Is this connection slated to be shared to? context=edit only
+ *     done:         (boolean) Is this post (or connection) done sharing? context=edit only
+ *     toggleable:   (boolean) Can the current user change the `enabled` setting for this Connection+Post? context=edit only
+ *   }
+ *   ...
+ *   meta: { # Not defined in this file. Handled in modules/publicize/publicize.php via `register_meta()`
+ *     jetpack_publicize_message: (string) The message to use instead of the post's title when sharing.
+ *   }
+ *   ...
+ * }
+ *
+ * @since 6.8.0
+ */
+class WPCOM_REST_API_V2_Post_Publicize_Connections_Field extends WPCOM_REST_API_V2_Field_Controller {
+	protected $object_type = 'post';
```

Comment:
This will always be overriden by the `register_fields()` call. Do we need it for anything before that?

Reply:
No - I left it from an earlier draft since I thought it made it more clear what the property was for.